infra/main.tf

```diff
@@ -0,0 +1,107 @@
+resource "azurerm_resource_group" "nachet-resources-group" {
+  name     = "nachet"
+  location = "canadacentral"
+}
+
+resource "azurerm_container_registry" "nachet-registry" {
+  name                = "registryNachet"
```

Comment:
nachetRegistry? nachetregistry to be consistent with your other names? are - not permitted? if so, what about underscores?

Reply:
Yes, "-" and "_" are not permitted here... I renamed it "aciacfiaregistry" because it's common infrastructure

Reply:
Then we also need to update the naming conventions ADR to explain.

For now, we can only speak from an ailab perspective (common infra for ailab products); it will be a long road until we have an org-wide registry and will require a proposal and approval to our counterparts (which we should do).